.github/workflows/cockpit-lib-update.yml

```diff
@@ -0,0 +1,31 @@
+name: cockpit-lib-update
```

Comment:
Do we want it? Will it work? Just making sure what's the status, no need to solve that now.

Reply:
Sorry I just saw this. I will make sure to test this today